1-js/99-js-misc/04-reference-type/article.md

```diff
@@ -62,53 +62,53 @@ let user = {
 }
 
 *!*
-// split getting and calling the method in two lines
+// Se divide la obtención y se llama al método en dos lineas
 let hi = user.hi;
-hi(); // Error, because this is undefined
+hi(); // Error porque this es indefinido
 */!*
 ```
 
-Here `hi = user.hi` puts the function into the variable, and then on the last line it is completely standalone, and so there's no `this`.
+Aquí `hi = user.hi` coloca la función dentro de una variables y luego la última linea es completamente independiente, por lo tanto no hay `this`.
 
-**To make `user.hi()` calls work, JavaScript uses a trick -- the dot `'.'` returns not a function, but a value of the special [Reference Type](https://tc39.github.io/ecma262/#sec-reference-specification-type).**
+**Para hacer que la llamada `user.hi()` funcione, JavaScript usa un truco: el punto `'.'` no devuelve una función, sino un valor especial del [Tipo de referencia](https://tc39.github.io/ecma262/#sec-reference-specification-type).**
 
-The Reference Type is a "specification type". We can't explicitly use it, but it is used internally by the language.
+El Tipo de Referencia un "tipo de especificación". No podemos usarla explícitamente pero es usada internamente por el lenguaje.
 
-The value of Reference Type is a three-value combination `(base, name, strict)`, where:
+El valor del Tipo de Referencia es una combinación de triple valor `(base, name, strict)`, donde:
 
-- `base` is the object.
-- `name` is the property name.
-- `strict` is true if `use strict` is in effect.
+- `base` es el objeto.
+- `name` es el nombre de la propiedad.
+- `strict` es verdadero si `use strict` se lleva a cabo.
```

Comment:
```suggestion
- `strict` es verdadero si `use strict` está en efecto.
```
No me parece "a cabo" como una accion que se "efectua"
Acá dudé, ¿se dice está en efecto? 
como: está " activo" o prendido, o actuando.

Reply:
Llevar a cabo, realizar... Here that means the same